ddp-workspace/projects/ddp-dsm-ui/src/app/file-download/file-download.component.ts

```diff
@@ -21,47 +22,79 @@ export class FileDownloadComponent implements OnDestroy {
   downloading = false;
   CLEAN = 'CLEAN';
   subscription: Subscription;
+  subscription2: Subscription;
 
-  constructor(private dsmService: DSMService ) {
+  constructor( private dsmService: DSMService ) {
 
   }
 
   ngOnDestroy(): void {
     if (this.subscription != null) {
       this.subscription.unsubscribe();
     }
+    if (this.subscription2 != null) {
+      this.subscription2.unsubscribe();
+    }
```

Comment:
it is possible to use multiple subscriptions in one Subscription instance. 
https://tinytip.co/tips/rxjs-subscription-add/

e.g. 
```
export class FileDownloadComponent {
  subscriptions = new Subscription();

  constructor(...) {...}

  downloadParticipantFile( file: File ): void {
    const subscription1 = this.dsmService.getSignedUrl(...);
    this.subscriptions.add(subscription1);
    ...
  }

  saveParticipantFile( url: any, type: string, fileName: string ): void {
     const subscription2 = this.dsmService.downloadFromSignedUrl( ... );
     this.subscriptions.add(subscription2);
     ...
  }

  ngOnDestroy(): void {
    this.subscriptions.unsubscribe();
  }
}

```